specs/proposals.md

```diff
@@ -62,6 +51,10 @@ If there is no newly finalized output, the service continues querying until it r
 output, and the appropriate timestamp, to the [L2 Output Commitment](#l2-output-commitment-smart-contract) contract's
 `appendL2Output()` function. The timestamp MUST be the next multiple of the `SUBMISSION_INTERVAL` value.
 
+> **Note regarding future work:** In the initial version of the system, the proposer will be the same entity as the
+> sequencer, which is a trusted role. In the future proposers will need to submit a bond in order to post L2 output
+> roots, and some or all of this bond may be taken in the event of a faulty proposal.
+
```

Comment:
Do we need to think through an upgrade plan to save ourselves a headache in the future?

Reply:
Yes, definitely (though not for this milestone). Should probably be part of a broader discussion on what sequencer decentralization is going to look like.

Reply:
Moved to a TLC issue